.vscode/launch.json

```diff
@@ -94,27 +94,6 @@
             "internalConsoleOptions": "openOnSessionStart",
             "outputCapture": "std"
         },
-        {
```

Comment:
I know we discussed it on gitter, but just for reference, I think we still need a way to launch individual test files from vscode so that we can easily debug them instead of launching them from the command line. You planned to add something similar in the future iirc? If so maybe add [WIP] to the PR just so we make sure it's still being worked on.

Reply:
> think we still need a way to launch individual test files from vscode so that we can easily debug them instead of launching them from the command line

Absolutely.

Also, running the tests in watch mode would be great. But that does not seem to be possible due to [this](https://github.com/inversify/InversifyJS/issues/563#issuecomment-323627172).

Reply:
I removed this configuration because it calls mocha for the entire workspace anyway. To use Jest with the entire workspace, install [vscode-jest](https://github.com/jest-community/vscode-jest).

Reply:
(Also, watch mode works completely fine for me, thanks to Jest's isolation. You may need to pass `--maxWorker` to forcibly enable it.)

Reply:
> I removed this configuration because it calls mocha for the entire workspace anyway.

That's not the case. It takes current files as arg.

Reply:
Okay, I'll add an identical profile for Jest later.

Reply:
> watch mode works completely fine for me

Currently, the watch mode has a problem. When it loads a module that has, for instance, the `decorate(injectable(), ...)` part, then it fails after trying to decorate the same constructor multiple times. Are you saying, it is not the case with `jest`; that would be great!